adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenSilentHandlerTest.java

```diff
@@ -767,8 +767,8 @@ private void clearCache(final ITokenCacheStore cacheStore) {
     }
 
     private AuthenticationRequest getAuthenticationRequest(final String authority, final String resource,
-                                                           final String clientId, final boolean IsExtendedLifetimeEnabled) {
-        final AuthenticationRequest authRequest = new AuthenticationRequest(authority, resource, clientId, IsExtendedLifetimeEnabled);
+                                                           final String clientId, final boolean isExtendedLifetimeEnabled) {
+        final AuthenticationRequest authRequest = new AuthenticationRequest(authority, resource, clientId, isExtendedLifetimeEnabled);
         return authRequest;
```

Comment:
ah? I'm not collapsing lines. Only thing changed is IsExtendedLife** to start with lower case. :)